Packs/DB2/Integrations/DB2/DB2.py

```diff
@@ -0,0 +1,400 @@
+
+""" HELPER FUNCTIONS """
+
+
+def clear(message: str):
+    """
+    Clean data with square brackets from message
+
+    Args:
+        message(str): Any message string
+
+    Returns:
+        string with clean message
+    """
+
+    def repl(x):
+        return ""
+
+    return (re.sub(CLEAN, repl=repl, string=message)).strip()
+
+
+def bind_variables(names: str, values: str) -> Any:
+    """
+    Binding of column names with their values or return list of values
+
+    Args:
+        names(str): column name to bind with values, must be in the length of the values list
+        values(str): the values to bind with columns, can be in the length of the names list
+
+    Returns:
+        Any: a dict with column and value as a key value pair or list of values
+    """
+    names_list = argToList(names)
+    values_list = argToList(values)
+
+    # assuming the order of values is correct
+    if values_list and not names_list:
+        return [val for val in values_list]
+    elif len(names_list) == len(values_list):
+        return dict(zip(names_list, values_list))
+    else:
+        raise Exception("The bind variables lists are not is the same length")
+
+
+""" COMMAND FUNCTIONS """
+
+
+def query_command(client: Client, args: Dict, *_) -> CommandResults:
+    """
+    Executes the db2 query with the connection that was configured in the Client
+
+    Args:
+        client(Client): the client object with db connection
+        args(demisto.args): arguments for the query-command
+    """
+    sql_query = str(args.get("query"))
+    limit = int(args.get("limit", 50))
+    skip = int(args.get("skip", 0))
+    bind_variable_name = args.get("bind_variables_name", "")
+    bind_variable_values = args.get("bind_variables_values", "")
+
+    try:
+        variables = bind_variables(bind_variable_name, bind_variable_values)
+        result, headers = client.execute_query(sql_query, variables)
+
+        converted_table = [dict(row) for row in result]
+        table = [{str(key): str(value) for key, value in dictionary.items()} for dictionary in converted_table]
+        table = table[skip: skip + limit]
+
+        human_readable = tableToMarkdown(name="Query result:", t=table, headers=headers, removeNull=True)
+
+        context = {"Result": table, "Query": sql_query, "DbName": f"{client.dbname}"}
+        client.close()
+
+        return CommandResults(
+            outputs_prefix="DB2",
+            outputs_key_field="Query",
+            outputs=context,
+            raw_response=result,
+            readable_output=human_readable,
+        )
+    except Exception as err:
+        client.close()
+        demisto.error(f"error:\n {err}")
+        return CommandResults(readable_output="No results found")
+
+
+def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], List[Any]]:
+    """
+    If the connection in the client was successful the test will return OK
+    if it wasn't an exception will be raised
+    """
+    return "ok", {}, []
```

Comment:
```suggestion
    return "ok"
```
Now when you use ```return_results``` you can just return the string ```ok```

Reply:
changed in [3fe28ed](https://github.com/demisto/content/pull/18139/commits/3fe28edaae14abdcd24bc1f718d86478d6bfd3ab)